src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: lighter;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.data-form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  background: url(images/Search.svg) no-repeat;
+  background-color: #fff;
+  color: #3d4e61;
+  font-weight: 900;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigInputSearch {
+  height: 70px;
+  line-height: 22px;
+  padding-left: 42px;
+  text-indent: 20px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position-x: 25px;
+  background-position-y: 24px;
+}
```

Comment:
You can add this values to `input` class, then just use `smallInputSearch` to overlap `input` styles, it's unnecessary to style both of them